package.json

```diff
@@ -9,6 +9,7 @@
     "@typescript-eslint/eslint-plugin": "^5.59.1",
     "@web/dev-server-esbuild": "^1.0.2",
     "@web/dev-server-import-maps": "^0.2.0",
+    "@web/dev-server-legacy": "^2.1.1",
```

Comment:
Seems small on the surface, but comes with a very significant dependency cost - just look at the "lines changed" count for package-lock.json!

Reply:
Okay, so I am _not_ keen on this. Yeah, it's a dev dependency, but it's gonna cost us significantly in dependabot and more. Adding dependencies needs to go through review and cost/benefit analysis.

Reply:
So... should I put this back in draft, pending such a review?  Or just abandon this and close out the issue as a `wontfix`?

We can't reasonably test against older Chrome versions without it, which kinda defeats our main reason to do remote testing against BrowserStack.

Reply:
It is relatively easy to [download older versions of Chromium](https://www.chromium.org/getting-involved/download-chromium/#downloading-old-builds-of-chrome-chromium) and test on those -- which I think would be sufficient. That then resolves the need to drag in 5,000 new lines of dependencies into package-lock.json and probably removes most of our BrowserStack requirement altogether?